CHANGELOG.md

```diff
@@ -20,6 +20,15 @@ Client Library | v0.10.1  | [symbol-openapi-typescript-fetch-client](https://www
 - Added `transferMosaicId`, `fromTransferAmount`, `toTransferAmount` to transaction searches.
 - Added `CurrencyService` to allow loading Network and custom `Currency` objects from the rest service.
 - Added `UnlockedAccount` endpoint in `NodeRepository` for checking delegated harvesting status.
+- Added `StateProofService` to verify the different states.
+- Added `serialize()` to state objects like `AccountInfo`, `MosaicInfo`, etc.
```

Comment:
in 
AccountInfor
MosaicInfo
NamespaceInfo
MultisigInfo
AccountRetrictionInfo
MosaicRestrictionInfo
MetadataEntry
SecretLockInfo
HashLockInfo

to generate state proof hash